ChangeLog.md

```diff
@@ -1,1095 +1,1118 @@
-   Content-MD5 hashes when uploading, you must now specify `--put-md5` on the command line.
-
-### New features
-
-1. Can migrate data directly from Amazon Web Services (AWS). In this high-performance data path
-   the data is read directly from AWS by the Azure Storage service. It does not need to pass through
-   the machine running AzCopy. The copy happens synchronously, so you can see its exact progress.  
-1. Can migrate data directly from Azure Files or Azure Blobs (any blob type) to Azure Blobs (any
-   blob type). In this high-performance data path the data is read directly from the source by the
-   Azure Storage service. It does not need to pass through the machine running AzCopy. The copy
-   happens synchronously, so you can see its exact progress.  
-1. Sync command prompts with 4 options about deleting unneeded files from the target: Yes, No, All or
-   None. (Deletion only happens if the `--delete-destination` flag is specified).
-1. Can download to /dev/null. This throws the data away - but is useful for testing raw network
-   performance unconstrained by disk; and also for validating MD5 hashes in bulk (when run in a cloud
-   VM in the same region as the Storage account)
-
-### Bug fixes
-
-1. Fixed memory leak when downloading large files
-1. Fixed performance when downloading a single large file
-1. Fixed bug with "too many open files" on Linux
-1. Fixed memory leak when uploading sparse files (files with big blocks of zeros) to Page Blobs and
-   Azure Files.
-1. Fixed issue where application crashed after being throttled by Azure Storage Service. (The
-   primary fix here is for Page Blobs, but a secondary part of the fix also helps with Block Blobs.)
-1. Fixed functionality and usabilty issues with `remove` command
-1. Improved performance for short-duration jobs (e.g. those lasting less than a minute)
-1. Prevent unnecessary error message that sometimes appeared when cancelling a job
-1. Various improvements to the online help and error messages.
-
-
-## Version 10.0.8:
-
-1. Rewrote sync command to eliminate numerous bugs and improve usability (see wiki for details)
-1. Implemented various improvements to memory management
-1. Added MD5 validation support (available options: NoCheck, LogOnly, FailIfDifferent, FailIfDifferentOrMissing)
-1. Added last modified time checks for source to guarantee transfer integrity 
-1. Formalized outputs in JSON and elevated the output flag to the root level
-1. Eliminated outputs to STDERR (for new version notifications), which were causing problems for certain CI systems
-1. Improved log format for Windows
-1. Optimized plan file sizes
-1. Improved command line parameter names as follows (to be consistent with naming pattern of other parameters):
-   1. fromTo -> from-to
-   1. blobType -> blob-type
-   1. excludedBlobType -> excluded-blob-type
-   1. outputRaw (in "list" command) -> output
-   1. stdIn-enable (reserved for internal use) -> stdin-enable
+
+# Change Log

```

Comment:
I think this diff looks big because of line ending changes. Once this release is out I'm going to fix this line ending issue once and for all. 

Please only review the 10.27.0 related stuff below.